From: Maximiliano Curia <maxy@debian.org>
Date: Sat, 7 Nov 2015 16:00:53 +0100
Subject: format-security

## Description: Avoid format-security errors
## Origin/Author: Maximiliano Curia <maxy@debian.org>
---
 src/dialogs.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: debian-oregano/src/dialogs.c
===================================================================
--- debian-oregano.orig/src/dialogs.c
+++ debian-oregano/src/dialogs.c
@@ -63,7 +63,7 @@ oregano_error_with_title (gchar *title,
 		GTK_DIALOG_MODAL,
 		GTK_MESSAGE_ERROR,
 		GTK_BUTTONS_OK,
-		span_msg->str);
+		"%s", span_msg->str);
 
 	gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_OK);
 
@@ -99,7 +99,7 @@ oregano_warning_with_title (gchar *title
 		GTK_DIALOG_MODAL,
 		GTK_MESSAGE_WARNING,
 		GTK_BUTTONS_OK,
-		span_msg->str);
+		"%s", span_msg->str);
 
 	gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_OK);
 
@@ -120,7 +120,7 @@ oregano_question (gchar *msg)
 		GTK_MESSAGE_QUESTION,
 		GTK_BUTTONS_OK,
 		GTK_BUTTONS_CANCEL,
-		msg);
+		"%s", msg);
 
 	gtk_dialog_set_default_response (GTK_DIALOG (dialog), GTK_RESPONSE_CANCEL);
 
